x/market/handler/handler_test.go

```diff
@@ -313,6 +333,15 @@ func TestCloseBidValid(t *testing.T) {
 	res, err := suite.handler(suite.ctx, msg)
 	require.NotNil(t, res)
 	require.NoError(t, err)
+
+	t.Run("ensure event created", func(t *testing.T) {
+		iev := testutil.ParseMarketEvent(t, res.Events[3:4])
+		require.IsType(t, types.EventBidClosed{}, iev)
+
+		dev := iev.(types.EventBidClosed)
+
+		require.Equal(t, msg.BidID, dev.ID)
```

Comment:
I don't think it's needed here.  If anything, we want to make sure that `EventCloseBid` is _not_ emitted.

Reply:
Sorry, I didn't get you. This is to ensure `EventBidClosed` emitted when valid bid is closed.

Reply:
Sorry, the diff made it look like this was in `TestCloseBidNonExisting`.  You're right - it belongs here.